float64[] joint_angles

# Poses (x,y,z,qx,qy,qz,qw) of links; [7]*(JointNum+1)
float64[] link_x

# Simulation time
float64 time

